templates/frontend/objects/article_details.tpl

```diff
@@ -227,6 +239,22 @@
 						{$article->getDatePublished()|date_format:$dateFormatShort}
 					</div>
 				</div>
+
+				{* Display article versions *}
+				{if $versioningEnabled && (@count($previousRevisions) > 1)}
+					{if $previousRevisions|@count > 0}
```

Comment:
why double check of the count (> 1 and > 0) ? -- it seems that it is not necessary....